contrib/_incr_version

```diff
@@ -1,6 +1,13 @@
 #!/bin/sh -eu
+
 old_version="$1"
 new_version="$2"
+
+if [ "$new_version" != "${new_version#v}" ]; then
```

Comment:
style nit: then goes on the next line, and the extra newline added after the shebang doesn't need to be there.

Reply:
Done ;) I beg to differ but understand and respect your choice :)
Based on your dotfiles scripts I also switched to tabs for indentation. Hope that's ok.